src/main/java/Duke.java

```diff
@@ -1,10 +1,67 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String chatBot = "Hello! I'm TONY\n"
+                + "What can I do for you?\n"
+                + "_________________________";
+        System.out.println(chatBot);
+        Task[] tasks = new Task[100];
+        int count = 0;
+        while(true) {
+            Scanner userInput = new Scanner(System.in);
```

Comment:
Scanner should be opened once outside the loop and closed properly to avoid resource leaks